.github/workflows/publish_dashboard.yml

```
@@ -61,3 +61,24 @@ jobs:
       - name: Deploy to GitHub Pages
         id: deployment
         uses: actions/deploy-pages@v4
+      - name: Notify Discord (optional)
+        if: ${{ secrets.DISCORD_WEBHOOK_URL != '' || secrets.DISCORD_PM_WEBHOOK_URL != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)